.gitvote.yml

```diff
@@ -0,0 +1,263 @@
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-consensus-node-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - Swift
+  hiero-sdk-swift-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-swift-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - Python
+  hiero-sdk-python-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-python-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # DID SDK Python Teams
+  hiero-did-sdk-python-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-did-sdk-python-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - Rust
+  hiero-sdk-rust-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-rust-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - Java
+  hiero-sdk-java-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-java-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - JavaScript
+  hiero-sdk-js-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-js-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - C++
+  hiero-sdk-cpp-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-cpp-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK Teams - Go
+  hiero-sdk-go-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-go-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # SDK TCK Teams
+  hiero-sdk-tck-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-sdk-tck-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # Gradle Conventions Teams
+  hiero-gradle-conventions-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-gradle-conventions-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # Solo Action Teams
+  hiero-solo-action-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-solo-action-maintainers"]
+    close_on_passing: false
+    exclude_team_maintainers: false
+
+  # Protobufs Teams
+  hiero-protobufs-maintainers-vote:
+    duration: 2w
+    pass_threshold: 51
+    allowed_voters:
+      teams: ["hiero-protobufs-maintainers"]
+    close_on_passing: false
```

Comment:
why is this always `false`?

Reply:
to allow people to continue to vote, or change votes, imho